src/ipa/rpi/common/ipa_base.cpp

```diff
@@ -494,8 +493,16 @@ void IpaBase::processStats(const ProcessParams &params)
 			if (minFrameDuration_ != maxFrameDuration_)
 				LOG(IPARPI, Error) << "Sync algorithm enabled with variable framerate. " << minFrameDuration_ << " " << maxFrameDuration_;
 			offset = syncStatus.frameDurationOffset;
-			if (!syncStatus.ready)
+
+			if (syncStatus.syncLag) {
```

Comment:
Is "sync lag" the different between when we wanted the "sync" to start, and when it actually happened (for instance, because of frame drops)? In which case, how would we report if the "lag" really is zero?